src/App.tsx

```diff
@@ -21,9 +31,13 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            onChange={(event) => {
+              setPerPage(+event.target.value);
+              setCurPage(1);
+            }}
           >
             <option value="3">3</option>
-            <option value="5">5</option>
+            <option value="5" selected>5</option>
```

Comment:
Not necessary to use the `selected` attribute directly on an `<option>`, since the `value` attribute is specified on the parent `<select>`.

Reply:
I dont understand this comment. If i reload page, the selceted option shoud be "5" as in the sample